TypeScriptReact.YAML-tmLanguage

```diff
@@ -0,0 +1,720 @@
+# Similar to TypeScript.YAML-tmLanguage except:
+#   - Remove type assertion (cast.expr.ts)
```

Comment:
OK. Thanks for investigating. Maybe just create 2 different syntax files then, and try to keep the differences grouped together so they can be easily compared for any divergence as we iterate. Thanks!